catalog/spatial/csw/spatial-csw-endpoint/src/test/java/org/codice/ddf/spatial/ogc/csw/catalog/endpoint/transformer/CswRecordMapperFilterVisitorTest.java

```diff
@@ -185,7 +184,7 @@ public void testVisitBeyond() {
 
   @Test
   public void testVisitBBox() {
-    BBOX filter = factory.bbox(attrExpr, 0, 0, 10, 20, "EPSG:4269");
+    BBOX filter = factory.bbox(attrExpr, 0, 0, 10, 20, "EPSG:4326");
```

Comment:
❓  do we have to test/account for EPSG:4329 still being used anywhere?

Reply:
It still works with 4269, but it now actually tries to convert it and runs it through the algorithms, which means 10 is no longer 10, it comes out as 9.9999999999867 or something and then you'd be trying to do an exact match in the test against floating point rounding across OSs. So I changed it to one that wouldn't get run through algorithms since it is the default projection.